knowledge-base/grid-custom-grouping-order.md

```diff
@@ -0,0 +1,188 @@
+---
+title: Custom Grouping Order in Grid for Blazor
+description: Learn how to implement custom grouping order in the Telerik Grid for Blazor.
+type: how-to
+page_title: Implementing Custom Grouping Order in Blazor Grid
+slug: grid-custom-grouping-order
+tags: grid, blazor, custom, grouping, order, sorting
+res_type: kb
+ticketid: 1661227
+---
+
+## Environment
+<table>
+    <tbody>
+	    <tr>
+	    	<td>Product</td>
+	    	<td>Grid for Blazor</td>
+	    </tr>
+    </tbody>
+</table>
+
+## Description
+
+I want to group data programmatically in the Telerik Grid for Blazor by one column, ensuring the grouped order matches a specific sequence rather than the default ascending or descending order. I need to sort groups in a custom way, based on a predefined order of the data received from the backend or a custom pattern.
```

Comment:
```suggestion
I want to group data programmatically in the Telerik Grid for Blazor by one column. The group order must match a specific sequence rather than the default ascending or descending order. I need to sort groups in a custom way, based on a predefined order of the data, or a custom pattern.
```